requirements.txt

```diff
@@ -27,3 +27,4 @@ ufolint==1.2.0
 unicodedata2==15.0.0
 vharfbuzz==0.2.0
 ufo2ft==2.32.0
+shaperglot>=0.2.0
```

Comment:
Please always use `==` on **requirements.txt**.
And `>=` only on **setup.py**, when needed.